src/server.es6.js

```diff
@@ -2,41 +2,62 @@ import React from 'react';
 import { App } from 'horse';
 
 class ServerReactApp extends App {
-  * injectBootstrap () {
-    var p = this.props;
+  injectBootstrap (format) {
+    return function * () {
+      var p = this.props;
+
+      if (format) {
+        p = format(p);
```

Comment:
Realistically, this is the furthest point in the render; but, I'll turn it into a copy to be safe, since this is a generator.